website/www/site/archetypes/blog.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "{{ replace .Name "-" " " | title }}"
+date: "{{ .Date }}"
+categories: blog
+authors: "Your Name"
```

Comment:
Please make it an array like here

![image](https://user-images.githubusercontent.com/15073128/79421161-8594fd80-7fba-11ea-9a84-fc19a209923e.png)